### DISCUSSION TITLE
Need to Run optimization   cases for validation of framework

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#142] [General]
Dear all,
I would like to replicate two optimization problems with SLSQP as optimizer for published ADODG cases.

1) Drag minimization of the NACA0012 in transonic inviscid flow
2) Drag minimization of RAE2822 in trnsonic visocus flow.
3) or any other optimization problem published in literature, so that performing it with dafoam framework and showing that the end result is similar to that of published optimized results.

Kindly guide me regarding this or if anyone has done this, please help me with meshscript files.

Regards,
Usman shahid

**Comment by friedenhe** [comment]
I suggest you try the IPOPT optimizer, which should be better than SLSQP. The DAFoam docker image comes with the IPOPT optimizer. Also, refer to the [FAQ](https://dafoam.github.io/mydoc_get_started_faq.html#how-to-use-a-finer-mesh) section on how to refine the mesh.

**Reply by usmanshahid90** [reply]
Alright. Thanks I will definitely try this out and will give a comparison between SLSQP and IPOPT.
